java/src/jmri/jmrit/operations/setup/Setup.java

```diff
@@ -175,15 +176,15 @@ public class Setup extends PropertyChangeSupport implements InstanceManagerAutoD
             COLOR, KERNEL, KERNEL_SIZE, OWNER, DIVISION, TRACK, LOCATION, DESTINATION, DEST_TRACK, FINAL_DEST, FINAL_DEST_TRACK,
             COMMENT, DROP_COMMENT, PICKUP_COMMENT, RWE };
     private static final String[] ENGINE_ATTRIBUTES = { ROAD, NUMBER, TYPE, MODEL, LENGTH, WEIGHT, CONSIST, OWNER,
-            TRACK, LOCATION, DESTINATION, COMMENT };
+            TRACK, LOCATION, DESTINATION, COMMENT, DCC_ADDRESS };
     /*
      * The print Manifest and switch list user selectable options are stored in the
      * xml file using the English translation.
      */
-    private static final String[] KEYS = { "Road", "Number", "Type", "Model", "Length", "Weight", "Load", "Load_Type",
-            "Color", // NOI18N
-            "Track", "Destination", "Dest&Track", "Final_Dest", "FD&Track", "Location", "Consist", "Kernel", // NOI18N
-            "Kernel_Size", "Owner", "RWE", "Comment", "SetOut_Msg", "PickUp_Msg", "Hazardous", "Tab", "Tab2", "Tab3" }; // NOI18N
+    private static final String[] KEYS = {"Road", "Number", "Type", "Model", "Length", "Weight", "Load", "Load_Type",
+            "Color", "Track", "Destination", "Dest&Track", "Final_Dest", "FD&Track", "Location", "Consist", "Kernel",
+            "DCC_Address", "Kernel_Size", "Owner", "RWE", "Comment", "SetOut_Msg", "PickUp_Msg", "Hazardous", "Tab",
+            "Tab2", "Tab3"};
```

Comment:
It would be good to add `// NOI18N`

Reply:
Is it needed per line, or just at the end of the strings?  In the past it was per line to get rid of the warnings.

Reply:
I wasn't aware that there is a tool that checked these. So I don't know.
Have you used a tool to check this? If so, which tool?

I noticed that `// NOI18N` was removed when I read the source code of the PR, but I have never used a tool that checked it.

Reply:
I was just looking for the ant tool that we used to use for checking. Years ago it was a requirement to have the // NOT18N notation or the commit would fail.  I don't believe we require it any more.  The issue, like the  string I have, is that the notation // NOT18N was needed per line which can screw up the formatting.

Reply:
I think it would be good to re-enable that tool again to ensure strings that should be translated are translated. But I don't know how to do it.

@bobjacobsen  What's your thoughts about this?